### DISCUSSION TITLE
Q: Does this plugin support major.minor.patch.hotfix format?

### DISCUSSION THREAD

**Discussion by vitalyk-multinarity** [PSanetra/git-semver#7] [Q&A]
Does this plugin support major.minor.patch.hotfix format - for instance, v1.2.3.4?

Thanks,
Vitaly

**Comment by PSanetra** [comment]
Hi @vitalyk-multinarity

as this plugin is intended to support the [semver](https://semver.org/) format, only a format like `v1.2.3-hotfix.1` would be supported, although the suffix like `-hotfix.1` is actually intended as a pre-release suffix.

I don't known any approach for hotfix cases in the semantic versioning standard. Dependening on your use case and ecosystem using the pre-release format could be ok for you, but be aware: If some tooling parses this format as valid semver it would prioritize `v1.2.3` over `v1.2.3-hotfix.1` as `v1.2.3-hotfix.1` might be interpreted as pre-release version and `v1.2.3` would be more recent according to semver standard semantically.

**Reply by vitalykarasik** [reply]
Thank you @PSanetra!
Our release manager chose to work with v1.2.3.4 format for releases, and you're right - it's not strictly semver convention.
